style: apply /style-guide pass to models/runs (#2724)
wandb/docs: models/runs/color-code-runs.mdx

Create visually meaningful line plots and plot legends by color-coding your W&B runs based on metrics or configuration parameters. Identify patterns and trends across experiments by coloring runs according to their performance metrics (highest, lowest, or latest values). W&B automatically groups your runs into color-coded buckets based on the values of your selected parameter.

> To use metric- or configuration-based colors for your runs, turn on key-based colors and then choose the metric or configuration key and Y value that determine each run's color.

Reviewer: ```suggestion
To use metric- or configuration-based colors for your runs, you need to configure two values a **Key** and a **Y value**. Use the key to specify the metric or config value you want to use. Use the Y value to specify the buckets W&B uses to assign colors.
```

Author: Applied your suggested rewording.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

The following sections describe how to enable key-based colors, choose a metric or configuration key, set a Y value, and customize the buckets W&B uses to assign colors.

## Turn on key-based colors

Enable key-based colors so W&B can assign run colors based on a metric or configuration key instead of the default per-run colors.

1. Navigate to your W&B project.
2. Select the **Workspace** tab from the project sidebar.
3. Click the **Settings** icon in the top right corner.
4. From the drawer, select **Runs**.
5. In the **Run colors** section, select **Key-based colors**.
6. Configure the following options:

>     - From the **Key** dropdown, select the metric you want to use for assigning colors to runs.

Reviewer: ```suggestion
    - From the **Key** dropdown, select the metric or config value you want to use for assigning colors to runs.
```

Author: Applied — now "the metric or config value".

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

    - From the **Y value** dropdown, select the Y value you want to use for assigning colors to runs.
    - Set the number of buckets to a value from 2 to 8.

<Note>
When you use key-based colors, the option to [customize run colors](/models/runs/run-colors) isn't available.
</Note>

> After you turn on key-based colors, W&B colors each run based on the key and Y value you selected. The following sections describe how to set the metric and Y value and how to customize the buckets that W&B uses to assign colors to runs.

Reviewer: ```suggestion
After you turn on key-based colors, W&B colors each run based on the key and Y value you selected. 

The following sections describe how to set the metric and Y value and how to customize the buckets that W&B uses to assign colors to runs.
```

Author: Applied — split into two paragraphs.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

## Set a configuration key

> As an alternative to using a metric, you can color runs based on a configuration key such as a hyperparameter. The configuration options in your **Key** dropdown come from the key-value pairs you pass to the `config` parameter when you initialize a W&B run. Use configuration keys to log hyperparameters or other settings used in your training or evaluation scripts.

Reviewer: ```suggestion
Color runs based on a configuration key such as a hyperparameter. The configuration options in your **Key** dropdown come from the key-value pairs you pass to the `config` parameter when you initialize a W&B run. Use configuration keys to log hyperparameters or other settings used in your training or evaluation scripts.
```

Author: Applied — dropped the "As an alternative" lead-in.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

Within the **Key** dropdown, `"learning_rate"`, `"batch_size"`, and `"optimizer"` are available options.

## Set a Y value

> After you select a key, set a Y value to control which value of the selected metric drives the color assignment. The Y value determines which value of the selected metric W&B uses to assign a color to each run. You can choose from the following options:

Reviewer: ```suggestion
After you select a key, set a Y value to control which value of the selected metric or config value drives the color assignment. The Y value determines which value W&B uses to assign a color to each run. You can choose from the following options:
```

Author: Applied your suggested rewording.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_
